bin/inx-chronicle/src/inx_listener.rs

```diff
@@ -58,12 +57,25 @@ impl Actor for InxListener {
     type Error = InxListenerError;
 
     async fn init(&mut self, cx: &mut ActorContext<Self>) -> Result<Self::State, Self::Error> {
-        info!("Connecting to INX...");
+        log::info!("Connecting to INX at bind address `{}`.", self.config.address);
```

Comment:
I would mention the bind address only once, preferably when it has succeeded, so in line 63.

Reply:
I kept the _connecting_ one, because it is easier to debug if something goes wrong.